.gitignore

```diff
@@ -5,6 +5,12 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
+# Ignore top-level built files
+forEachVariation.js
```

Comment:
does `/forEachVariation.js` mean that we don't need to unignore src?

Reply:
yup! nice